client/utils/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import _ from 'lodash'
 import ProductCard from '../components/ProductCard'
+import {compose} from 'redux'
```

Comment:
are you using this anywhere?

Reply:
No...not sure what that's from. Maybe it was imported automatically through a typo